### PR TITLE
Challenge 2

### PR DESCRIPTION
**Pull request by daktari01**
### What does this PR do?
1. Registers user to WeConnect
2. Logs in the user to WeConnect given the correct username and password.
3. Registers a business
4. Gets either a single business by business id or all registered businesses
5. Edits a particular businesses
6. Posts a review for a business
7. Gets reviews for that business
8. Deletes a business
9. Resets user password
10. Logs user out.

### Description of Task to be completed?
Undertake [challenge 2](https://docs.google.com/document/d/1iJrCZKiHl-9bIqsTinHmipI_ItYZgrPHT2ZzO3W7ACg/edit) of WeConnect application

### How should this be manually tested?
On Postman, use the various HTTP methods to test the endpoints as described in the [ReadMe](https://github.com/daktari01/we_connect/blob/development/README.md)

### What are the relevant pivotal tracker stories?
- [WeConnect Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2153318)

### Any background context you want to add?
- This is version 1 of WeConnect.

### Screenshots
- N/A

### CODE REVIEW

**Comment by mwenda-eric** [comment]
Looks good, All endpoints working as expected

**Comment by ThaDeveloper** [comment]
Great PR, great coding standard.Proceed

**Comment by victorjambo** [comment]
Looks good

File: tests/test_weconnect.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        self.assertEqual(response.status_code, 200)
+        view_response = self.client().get('/api/v1/businesses/1/reviews', 
+            headers={'content-type':'application/json', 
+                'x-access-token':self.token})
+        self.assertEqual(view_response.status_code, 200)
+
+    def tearDown(self):
+        self.business_i.businesses.clear()
+        self.test1_user.clear()
+        # self.reset_user.clear()
```
Comment (jm-wanja) [inline tests/test_weconnect.py]: Remove unused code if you don't need it, before merging to master.

File: tests/test_weconnect.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            data=json.dumps(self.login_user), 
+            headers={'content-type':'application/json'})
+        self.client().post('/api/v1/auth/login', 
+            data=json.dumps(self.login_user), content_type='application/json')
+        response = self.client().post('/api/v1/auth/reset-password', 
+            data=json.dumps(self.reset_passw), 
+            headers={'content-type':'application/json', 
+                'x-access-token':self.token})
+        self.assertEqual(response.status_code, 200)
+        # self.assertIn("Password reset successful", str(response.data))
```
Comment (jm-wanja) [inline tests/test_weconnect.py]: Remove commented code if not needed.

**Comment by jm-wanja** [comment]
Good job!
Look into validating the input  field, so that a user cannot post an empty review text or business details.

File: tests/test_weconnect.py (hunk 2 above)
Comment (daktari01) [reply tests/test_weconnect.py]: Thanks, @jm-wanja  for the review. Working on it straight away.

**Pull request merged**